assets/helpers/abTests/__tests__/abtestTest.js

```diff
@@ -118,6 +118,169 @@ describe('basic behaviour of init', () => {
     expect(participations).toEqual(expectedParticipations);
   });
 
+  it('The ab test framework should check for both (min and max) breakpoints if they are provided', () => {
+
+    window.matchMedia = window.matchMedia || jest.fn(() => ({ matches: false }));
+    document.cookie = 'GU_mvt_id=12346';
```

Comment:
Could you use `beforeEach` for these?

Reply:
I didn't put it in a beforeEach because it is not used in all the test, but I think it would not harm other tests to have this function defined.

Reply:
Move the function to `beforeEach` here: 126b71260eec55e8528a369bc3bba3da9c7bb710